crates/wash-cli/tests/common/mod.rs

```diff
@@ -73,6 +73,21 @@ pub fn test_dir_file(subfolder: &str, file: &str) -> PathBuf {
     PathBuf::from(format!("{root_dir}/tests/fixtures/{subfolder}/{file}"))
 }
 
+#[allow(unused)]
+/// writes content to specified file path... creates file if it doesn't exist
+pub async fn set_test_file_content(path: &PathBuf, content: &str) -> Result<()> {
+    let mut file = File::create(path).await.context(format!(
+        "failed to open/create test file {}",
+        path.to_string_lossy()
+    ))?;
+
+    file.write_all(content.as_bytes()).await.context(format!(
+        "failed to write content to test file {}",
+        path.to_string_lossy()
+    ))?;
+    Ok(())
+}
+
```

Comment:
is there are particular reason why to `tokio` async variant is wanting here?

Reply:
Hey ahmed I think you were trying to note that [`tokio::fs::write`](https://docs.rs/tokio/latest/tokio/fs/fn.write.html) is preferrable, that's totally right, please use that! that is the better option since it's in an `async fn`